src/main/java/com/example/springfoundation/entity/TestObject.java

```diff
@@ -0,0 +1,17 @@
+package com.example.springfoundation.entity;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+@Entity
+@Table(name = "test_table")
+@Data
```

Comment:
Be careful with `@Data` annotation on **Entities**
https://jpa-buddy.com/blog/lombok-and-jpa-what-may-go-wrong/

Reply:
Noted. Removed lombok.Data from entity. Replaced it with implementation provided by JPABuddy.